code/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "start": "webpack serve --config ./config/webpack.config.js --mode development",
-    "build": "webpack --config ./config/webpack.config.js --mode production",
+    "start": "webpack serve --config ./config/webpack.development.config.js --mode development",
+    "start:prod": "webpack serve --config ./config/webpack.production.config.js --mode production",
+    "start:test": "webpack serve --config ./config/webpack.test.config.js --mode development",
+    "build": "webpack --config ./config/webpack.production.config.js --mode production",
```

Comment:
I'm curious why we use `--mode` flag here. I would be glad to hear more details about it

Reply:
It specifies mode argument that is passed to webpack configuration in case we create configuration as function. I guess that argument was not used before and is not used now inside of configuration. I just left it as it was